sound/soc/sof/topology.c

```diff
@@ -2537,7 +2549,15 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	}
 
 	w->dobj.private = swidget;
-	list_add(&swidget->list, &sdev->widget_list);
+
+	/* process components are added to widget_list in sof_process_load() */
+	if (w->id != snd_soc_dapm_effect) {
+		list_add(&swidget->list, &sdev->widget_list);
+		mutex_lock(&swidget->use_count_mutex);
+		swidget->use_count++;
```

Comment:
use_count++ for widget_ready ? so its initialized value == 1 ?

Reply:
not applicable in the new patchset.